src/components/user-list.html

```diff
@@ -1,24 +1,60 @@
-<link rel="import" href="./user-profile.html">
+<link rel="import" href="../../node_modules/@banno/polymer/polymer.js">
+<link rel="import" href="../../node_modules/@banno/polymer/lib/elements/dom-repeat.js">
 <link rel="import" href="./user-profile.html">
 
 <dom-module id="user-list">
-    
-    <template>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
+  <template>
+    <style>
+      .centered {
+        text-align: center;
+      }
+
+      .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+        border-radius: 8px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
+    </style>
+
+    <template is="dom-repeat" items="[[users]]" as="user">
+      <div>
+        <user-profile user="[[user]]" storage="[[storage]]"></user-profile> 
+      </div> 
+    </template> 
+    <button class="button" on-click="deleteAllUsers">Delete AllUsers</button>
     </template>
 
     <script type="module">
-        import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
-        
-        class UserList extends PolymerElement {
-            static get is() { return 'user-list'; }
-        }
-
-        customElements.define(UserList.is, UserList);
+      import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    
+    class UserList extends PolymerElement {
+      static get is() { return 'user-list'; }
 
-        export default UserList;
-    </script>
+      static get properties() {
+        return {
+          users: Object,
+          storage: Object
+        };
+      }
 
+      deleteAllUsers() {
+        this.storage.deleteAllUsers();
+        window.location.reload();
```

Comment:
what's the purpose of calling `reload()`? I really don't think you should need to use it... about ever.

Reply:
Just have this here so a new user list is automatically generated to show the changes made vs having to reload the page. The data is being updated in localStorage in real time it just requires a reload to update the user cards in the list to reflect the change made in localStorage.